3-UsingAPIs/Week1/homework/ex2-checkDoubleDigits.js

```diff
@@ -12,8 +12,13 @@ Complete the function called `checkDoubleDigits` such that:
   "Expected a double digit number but got `number`", where `number` is the 
   number that was passed as an argument.
 ------------------------------------------------------------------------------*/
-function checkDoubleDigits(/* TODO add parameter(s) here */) {
-  // TODO complete this function
+function checkDoubleDigits(number) {
+  return new Promise((resolve, reject) => {
+    if (number >= 10 && number <= 99)
+      resolve('This is a double digit number!')
+    else
+      reject(new Error(`Expected a double digit ${number}`))
```

Comment:
It is a best practice to always use curly braces around if...else blocks. Also, end statements with a semi-colon. Normally  Prettier would do add semicolons automatically if you forgot to do it yourself.

```suggestion
    if (number >= 10 && number <= 99) {
      resolve('This is a double digit number!');
    } else {
      reject(new Error(`Expected a double digit ${number}`));
    }
```

Reply:
Apart from the formatting, the code works fine.

Reply:
I added curly braces and semi colons 👍